guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

```diff
@@ -97,6 +97,13 @@ Ensure {SmartProxy} has access to `{RepoRHEL8ServerSatelliteMaintenanceProductVe
 # grep foreman_url /etc/foreman-proxy/settings.yml
 ----
 
+. Enable maintenance module:
++
+[options="nowrap" subs="attributes"]
+----
+# dnf module enable satellite-maintenance:el8
```

Comment:
While enabling the maintenance module, is the command same for both - `Satellite` and ` Capsule` ?

https://github.com/theforeman/foreman-documentation/blob/0108a6d32eea03598f953c56ef8239f77b0fe96c/guides/doc-Upgrading_and_Updating/topics/upgrading_a_connected_satellite_server.adoc?plain=1#L50

Reply:
@mjivraja yep, we are only enabling the dnf module.

Reply:
Ack!